core/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2018"
 [features]
 alloc = [ "linked_list_allocator" ]
 custom_panic_handler = []
+custom_allocator = []
```

Comment:
I think the feature should be renamed as it will not remove the `linked_list_allocator` as a dependency. As far as I understand the flag is intended to be used a some kind of allocation interception mechanism for debugging. So, the feature could be called `allocator_interception`?

Reply:
Resolved by the use of a `heap` feature instead.